tests/Behat/Context/Cli/InstallingPluginContext.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\RbacPlugin\Behat\Context\Cli;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+use Sylius\RbacPlugin\Cli\InstallPluginCommand;
+use Sylius\RbacPlugin\Entity\AdministrationRoleInterface;
+use Sylius\RbacPlugin\Model\PermissionInterface;
+use Sylius\RbacPlugin\Provider\SyliusSectionsProviderInterface;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Tests\Application\RbacPlugin\Entity\AdminUser;
+use Webmozart\Assert\Assert;
+
+final class InstallingPluginContext implements Context
+{
+    /** @var KernelInterface */
+    private $kernel;
+
+    /** @var Application */
+    private $application;
+
+    /** @var CommandTester */
+    private $tester;
+
+    /** @var InstallPluginCommand */
+    private $command;
+
+    /** @var RepositoryInterface */
+    private $administratorRepository;
+
+    /** @var RepositoryInterface */
+    private $administrationRoleRepository;
+
+    /** @var SyliusSectionsProviderInterface */
+    private $syliusSectionsProvider;
+
+    public function __construct(
+        KernelInterface $kernel,
+        RepositoryInterface $administratorRepository,
+        RepositoryInterface $administrationRoleRepository,
+        SyliusSectionsProviderInterface $syliusSectionsProvider
+    ) {
+        $this->kernel = $kernel;
+        $this->administratorRepository = $administratorRepository;
+        $this->administrationRoleRepository = $administrationRoleRepository;
+        $this->syliusSectionsProvider = $syliusSectionsProvider;
+    }
+
+    /**
+     * @When I install RBAC plugin by specifying root administrator's email as :administratorEmail
+     */
+    public function installRbacPlugin(string $administratorEmail): void
+    {
+        $this->application = new Application($this->kernel);
+        $this->application->add(new InstallPluginCommand($this->syliusSectionsProvider, $administratorEmail));
```

Comment:
We can leave it this way, but I'm wondering, could we define an `Application` service with already added `InstallPluginCommand` command and inject this service into the context... 🤔